package-lock.json

```diff
@@ -1,4574 +1,8 @@
 {
 	"name": "decorator-sample",
 	"version": "0.0.1",
-	"lockfileVersion": 2,
+	"lockfileVersion": 1,
```

Comment:
Do you have the latest node LST version installed?

This lock file version shouldn't have changed.